extensions/sql-migration/src/wizard/wizardController.ts

```diff
@@ -81,9 +84,44 @@ export class WizardController {
 		await Promise.all(wizardSetupPromises);
 		await pages[0].onPageEnter();
 
-		this.extensionContext.subscriptions.push(wizard.doneButton.onClick(async (e) => {
+		this.extensionContext.subscriptions.push(this._wizardObject.doneButton.onClick(async (e) => {
 			await stateModel.startMigration();
 		}));
+
+		this._wizardObject.cancelButton.onClick(e => {
+			sendSqlMigrationActionEvent(
+				TelemetryViews.SqlMigrationWizard,
+				TelemetryAction.PageButtonAction,
+				{
+					'sessionId': this._model._sessionId,
```

Comment:
I assume you want the session ID for this as well? Because we already track click events on wizard pages. But if you have custom properties then yeah I guess this is necessary.

Reply:
Yeah, we need the session id for these events